ewatercycle/models/abstract.py

```diff
@@ -42,6 +42,16 @@ def __del__(self):
         except AttributeError:
             pass
 
+    def __str__(self):
```

Comment:
Nice to see `__str__` methods, Can tests be added for them?

Reply:
Test are added to marrmot and wflow tests.